boat-scaffold/src/main/templates/boat-angular/package.mustache

```diff
@@ -15,18 +15,18 @@
     "build": "ng-packagr -p ng-package.json"
   },
   "peerDependencies": {
-    "@angular/core": "^{{ngVersion}}",
+    "@angular/core": ">={{ngVersion}}",
     {{#withMocks}}
     "@backbase/foundation-ang": "^{{foundationVersion}}",
     {{/withMocks}}
     "rxjs": "^{{rxjsVersion}}"
```

Comment:
Do we want to do the same with rxjs and foundation-ang?

Reply:
@daiscog we want to support rxjs 6 and 7 too right ?